### DISCUSSION TITLE
SSO sign with gmail is not available while signing in to nebari.qunsight.dev

### DISCUSSION THREAD

**Discussion by MekongDelta-mind** [nebari-dev/nebari#1843] [Q&A]
First of all, thanks for providing such good platform with reasonable resources for everyone who can't invest in high end devices. I got to know about this site while one of your Engineer was explaining about the usage of Dask in Nebari in PyCon US 2023. And from then on I have been using the and exploring the platform. It has been a very smooth ride with small hiccups in between .

Now why I;m writing this is because I'm unable to access my account since today( 24-06-2023, 10:25 IST). Previously there was a button for signing in with Google SSO, but the login page doesn't have the SSO anymore.

Have the team removed the SSO option from the login page?
Page 1
![image](https://github.com/nebari-dev/nebari/assets/47595700/a5103566-17c2-4eca-93f2-a852e8d57196)

Page 2 (which had the SSO sign in )
![image](https://github.com/nebari-dev/nebari/assets/47595700/879dd7f8-5da4-4a0e-bf91-da15f072d182)

**NOTE:** I do understand that this kind of great service can't stay free for long as it is too good to be true and there has to some revenue sources to pay for the systems running behind to keep the platform alive. So I do respect if you only allow the subscribed users now. Although I was working on some files yesterday and I didn't take local backup .  It would be great if you could please provide me with an alternative strategy to download my files, to work on it locally. Otherwise, I have to work with them from scratch.

Thanks for providing the platform.

**Comment by iameskild** [comment]
Hi @MekongDelta-mind, thank you for taking an interest in Nebari. Unfortunately, this was an oversight on our end; this specific Nebari instance should not have been open to the public. 

There might be a Nebari instance available to use in the future but at this time, we don't have anything setup - feel free to checkout [Nebari services](https://quansight.com/nebari-services/) (offered by Quansight) if you would like assistance and support setting up your own Nebari instance or try setting it up yourself by following [the docs](https://nebari-docs.netlify.app/docs/welcome).

Thanks again for interest in Nebari :)
